lib/ViewModels/FeatureInfoPanelSectionViewModel.js

```diff
@@ -82,7 +83,17 @@ function htmlFromFeature(viewModel, clock) {
     var feature = viewModel.feature;
     var data = viewModel.feature.properties;
     if (defined(viewModel.template)) {
-        return readTemplate(data, viewModel.template);
+        // template may be a string, eg. '<div>{{{Foo}}} Hello {{name}}</div>'
+        if (typeof viewModel.template === 'string') {
+            return Mustache.render(viewModel.template, data);
+        } else {
+            // or template may be an object with a main 'template' key, and additional partials
+            // eg. {template: '<div>test {{>foobar}}</div>', foobar: '<b>{{Foo}}</b>'}
+            var template = viewModel.template.template;
+            var partials = viewModel.template;
+            delete partials.template;
```

Comment:
This `htmlFromFeature` can be called multiple times, right?  So deleting this here will mean it's missing the next time around.